src/main/resources/db/migration/all/20240105113951__user_role_assignments_unique_by_role_and_user_id.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE user_role_assignments
```

Comment:
This whole approach relies on this constraint being reasonable and succeeding on dev.

Whilst I remove the seeded dev users, if users can really have multiple of the same roles this will not deploy and will need reverting.